documentation/synthetic_tokens/explainer.md

```diff
@@ -39,7 +39,7 @@ After the contract is deployed, anyone can interact with the contract to create
 
 Token sponsors deposit collateral into the contract to collateralize synthetic tokens, which they can then withdraw and trade with others. The first token sponsor to create synthetic tokens is able to immediately withdraw synthetic tokens from the contract. Any following token sponsors who wish to create synthetic tokens can do so, but must collateralize their positions by at least as much as the system’s [global collateralization ratio](glossary.md#global-collateralization-ratio-gcr). This ensures that all newly created tokens sponsor positions are correctly collateralized upon initialization. 
 
-The global collateralization ratio (GCR) is the average collateralization ratio among all token sponsor positions that have not yet been liquidated nor have any outstanding withdrawal requests. Requiring new token sponsors to collateralize their positions by as much as the GCR provides some assurances that so long as those token sponsors collateralized below the GCR have not yet been liquidated, those above the GCR need not be at risk of liquidation.
+The global collateralization ratio (GCR) is the average collateralization ratio among all token sponsor positions that have not yet been liquidated. Requiring new token sponsors to collateralize their positions by as much as the GCR provides some assurances that so long as those token sponsors collateralized below the GCR have not yet been liquidated, those above the GCR need not be at risk of liquidation.
```

Comment:
```suggestion
The global collateralization ratio (GCR) is the average collateralization ratio among all token sponsor positions that have not been liquidated. Requiring new token sponsors to collateralize their positions by as much as the GCR provides some assurances that so long as those token sponsors collateralized below the GCR have not yet been liquidated, those above the GCR need not be at risk of liquidation.
```

Reply:
Small nit: I just dropped the word "yet" because it implies that all sponsor positions will ultimately get liquidated, which isn't right.